includes/class-kadence-blocks-prebuilt-library-rest-api.php

```diff
@@ -600,6 +607,46 @@ public function get_remaining_credits( $request ) {
 			return rest_ensure_response( $response );
 		}
 	}
+	/**
+	 * Handle the event and maybe send to Prophecy WP.
+	 */
+	public function handle_event( $request ) {
+		$event_label = $request->get_param( self::PROP_EVENT_LABEL );
+		$event_data  = $request->get_param( self::PROP_EVENT_DATA );
+		$event_data  = json_decode( $event_data, true );
+
+		$event       = '';
+		$context     = array();
+
+		switch ( $event_label ) {
+			case 'ai_wizard_started':
+				$event = 'ai_wizard_started';
+				break;
+
+			case 'ai_wizard_save':
+				$event = 'ai_wizard_save';
+				$context = [
+					'organization_type' => $event_data['entityType'] ?? '',
+					'location_type'     => $event_data['locationType'] ?? '',
+					'location'          => $event_data['location'] ?? '',
+					'industry'          => $event_data['industry'] ?? '',
+					'mission_statement' => $event_data['missionStatement'] ?? '',
+					'keywords'          => $event_data['keywords'] ?? '',
+					'tone'              => $event_data['tone'] ?? '',
+					'collections'       => $event_data['customCollections'] ?? '',
+				];
+				break;
+		}
+
+		if ( $event ) {
```

Comment:
Let's do a little more nuanced checking here.

ChrisK [shared this article with me](https://www.beberlei.de/post/when_to_use_empty_in_php_i_say_never) a little while ago and while you're not using `empty()` here, I think the suggestions are still helpful.

```php
// Replace
if ( $event ) { }

// With
if ( strlen($string) === 0 ) { }
```

It allows for strict type checking and better communicates the type to whoever is reviewing this in the future.